src/ExportEdiJobs/ExportEdiJobDetailsActionMenu/ExportEdiJobDetailsActionMenu.js

```diff
@@ -0,0 +1,83 @@
+import { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { pick } from 'lodash';
+
+import { IfPermission } from '@folio/stripes/core';
+import {
+  Button,
+  Icon,
+  MenuSection,
+} from '@folio/stripes/components';
+import { useShowCallout } from '@folio/stripes-acq-components';
+
+import { useExportJobScheduler } from '../../common/hooks';
+import { useNavigation } from '../../hooks';
+
+export const ExportEdiJobDetailsActionMenu = ({
+  exportJob,
+  onToggle,
+  refetchJobs,
+}) => {
+  const { scheduleExportJob } = useExportJobScheduler();
+  const { navigateToEdiJobDetails } = useNavigation();
+  const showCallout = useShowCallout();
+
+  const jobParams = pick(exportJob, ['type', 'exportTypeSpecificParameters']);
```

Comment:
do it inside onRerun

with this line your useCallback doesn't make sense

Reply:
updated